src/workspace-hack/Cargo.toml

```diff
@@ -42,7 +44,8 @@ futures-io = { version = "0.3", features = ["std"] }
 futures-sink = { version = "0.3", features = ["alloc", "std"] }
 futures-task = { version = "0.3", default-features = false, features = ["alloc", "std"] }
 futures-util = { version = "0.3", features = ["alloc", "async-await", "async-await-macro", "channel", "futures-channel", "futures-io", "futures-macro", "futures-sink", "io", "memchr", "sink", "slab", "std"] }
-hashbrown = { version = "0.12", features = ["ahash", "inline-more", "nightly", "raw"] }
+hashbrown-594e8ee84c453af0 = { package = "hashbrown", version = "0.13", features = ["ahash", "inline-more"] }
+hashbrown-5ef9efb8ec2df382 = { package = "hashbrown", version = "0.12", features = ["ahash", "inline-more", "nightly", "raw"] }
```

Comment:
Is it possible to unify them?

Reply:
Looks like both versions have many crates depending on them. It is not likely to be unified.

Reply:
I've created multiple PRs but was pessimistic about that.